drivers/i2c/Kconfig

```diff
@@ -3,6 +3,9 @@ menu "Synopsys DesignWare I2C support"
 menuconfig DW_I2C
 	bool "Support DW I2C"
 
+config DW_I2C_CLK
+	int "Clock frequency (kHz)"
+	default 125000
```

Comment:
We normally prepare a definition, say DW_I2C_FREQ.
And define it in asm/mach/i2c.h before including driver/dw_i2c.h.